### PR TITLE
VACMS-4056: Add logic to restrict elements to certain roles on centralized content.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #4056

## Testing done
Visual

## Screenshots
image

## QA steps
As an administrator, visit `/node/16142/edit` and visually verify that deletes, actions (the three vertical dots elements), row drags, and `Show row weights` action link appear.
Log in as a user who is not a content admin / admin, visit `/node/16142/edit` and visually verify that deletes, actions (the three vertical dots elements), row drags, and `Show row weights` action link do not appear.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by ndouglas** [comment]
unrelated but nevertheless poetic 

Screen Shot 2021-06-28 at 1 46 52 PM

**Comment by ndouglas** [comment]
Still seeing row weights with a [normal user](https://cms-hh5yokv1sc3susjvaaawvlgutl47bs36.ci.cms.va.gov/user/2386/edit):

Screen Shot 2021-06-28 at 1 51 10 PM

I explicitly gave this user access to the Vet Centers section.

Prior to logging in with this account, I clicked "Show row weights" with my personal account.  When logging in I noticed that row weights were shown.  Perhaps that's a confounding factor, or perhaps I misread something.  Happy to assist further if needed but I'll be in backlog refinement for a couple hours now.

**Comment by ethanteague** [comment]
> Still seeing row weights with a [normal user](https://cms-hh5yokv1sc3susjvaaawvlgutl47bs36.ci.cms.va.gov/user/2386/edit):
> 
> Screen Shot 2021-06-28 at 1 51 10 PM
> 
> I explicitly gave this user access to the Vet Centers section.
> 
> Prior to logging in with this account, I clicked "Show row weights" with my personal account. When logging in I noticed that row weights were shown. Perhaps that's a confounding factor, or perhaps I misread something. Happy to assist further if needed but I'll be in backlog refinement for a couple hours now.

Thanks, @ndouglas ! I'll see what's up and ping you when I get it figured out.

File: docroot/modules/custom/va_gov_backend/js/script.es6.js [hunk 1]
```diff
@@ -34,6 +34,52 @@
     },
   };
 
+  /**
+   * Behaviors for hiding elements when user shouldn't have access.
+   * */
+  Drupal.behaviors.nationalDataHideDelete = {
+    attach(context) {
+      const adminRoles = ["content_admin", "administrator"];
```
Comment (kevwalsh) [inline docroot/modules/custom/va_gov_backend/js/script.es6.js:42]: This should only be administrator.

**Review by kevwalsh — CHANGES_REQUESTED** [review]
Tested as user/1523 who has access to /node/16142/edit as a content publisher, and this works well! 

But we also have to remove access for content admins, such as user/1342

**Comment by kevwalsh** [comment]
Ah, i missed the part about QAing the row weights, but i'm glad Nate caught that!

**Comment by ethanteague** [comment]
@kevwalsh & @ndouglas pr updated - have a look when you all get a sec, and I'll merge if it looks good.

**Comment by ndouglas** [comment]
Screen Shot 2021-06-29 at 8 51 41 AM

Looking into this test failure...

Screen Shot 2021-06-29 at 8 52 09 AM

Yup.

I'm not sure why this is happening, but lemme look into it and figure out what's up so your tests will pass.

**Review by ndouglas — APPROVED** [review]
LGTM, verified the elements are restricted appropriately.

**Pull request merged**